stdlib/public/Concurrency/Task.swift

```diff
@@ -76,42 +90,34 @@ public struct Task<Success, Failure: Error>: Sendable {
 
 @available(SwiftStdlib 5.5, *)
 extension Task {
-  /// Wait for the task to complete, returning (or throwing) its result.
+  /// Wait for the task to complete, returning its result or throwing an error.
   ///
-  /// ### Priority
-  /// If the task has not completed yet, its priority will be elevated to the
-  /// priority of the current task. Note that this may not be as effective as
-  /// creating the task with the "right" priority to in the first place.
+  /// If the task hasn't completed,
+  /// its priority increases to that of the current task.
+  /// Note that this might not be as effective as
+  /// creating the task with the correct priority,
+  /// depending on the executor's scheduling details.
   ///
-  /// ### Cancellation
-  /// If the awaited on task gets cancelled externally the `get()` will throw
-  /// a cancellation error.
+  /// If the task throws an error, this method propogates that error.
```

Comment:
```suggestion
  /// If the task throws an error, this computed property propogates that error.
```

I guess?

Reply:
Changed 'method' to 'property' here and in another spot in 946c72cbc73.